react/components/PriceRange/index.js

```diff
@@ -42,6 +42,7 @@ const PriceRange = ({ title, facets, priceRange, priceRangeLayout }) => {
       })
 
       setRange([left, right])
+      window.scroll({ top: 0, left: 0, behavior: 'auto' })
```

Comment:
could you use the `scrollToTop` prop here? so the user can set the behavior he prefers instead of always being `auto`. also, this prop allows the store to choose not to scroll when changing the price range

Reply:
@thalytafabrine I did some tests when I went to add this code and I realized that the scrollToTop prop is not working. I did tests in two different environments.

Workspace used: 
https://daniloscroll--acctglobal.myvtex.com/teste-lange/composicao_100%40perc%40%20Poli%C3%A9ster?initialMap=c&initialQuery=teste-lange&priceRange=0%20TO%2027&searchState=

Code in ws: https://prnt.sc/115uydh

Reply:
this is probably because the `scrollToTop` is set as `none` on your workspace. can you test using `scrollToTop` as `auto` or `smooth`?
![image](https://user-images.githubusercontent.com/20840671/113873623-5c511c80-978b-11eb-8c80-114d272aa5c2.png)

Reply:
@thalytafabrine But when setting the prop's value to none, wasn't this behavior (to scroll to the top) to cease to exist?

Reply:
yes, but in your workspace the `preventRouteChange` is set to `false`. this prop would only work with `preventRouteChange` as `true`, since when changing the route to another page, this new page will always be displayed from the top